### PR TITLE
Create sports-health.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/90674/sport-health#latest

Unsure about this one. Please review. :)

### CODE REVIEW

File: sports-health.csl [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    </sort>
+    <layout delimiter="," vertical-align="sup">
+      <text variable="citation-number"/>
+      <group prefix="(" suffix=")">
+        <label variable="locator" form="short" strip-periods="true"/>
+        <text variable="locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="false" et-al-min="7" et-al-use-first="3" second-field-align="flush">
```
Comment (adam3smith) [inline sports-health.csl:109]: definitely needs to be sorted alphabetically

File: sports-health.csl [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+          <text macro="title"/>
+        </substitute>
+      </names>
+    </group>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="article-newspaper" match="none">
+        <choose>
+          <if variable="DOI">
```
Comment (adam3smith) [inline sports-health.csl:46]:
```suggestion
          <if variable="PMID">
```

File: sports-health.csl [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+            <group delimiter=";" prefix=" ">
+              <choose>
+                <if variable="issue volume" match="any">
+                  <date variable="issued">
+                    <date-part name="year"/>
+                  </date>
+                </if>
+                <else>
+                  <group delimiter=" ">
+                    <text value="Published online"/>
```
Comment (adam3smith) [inline sports-health.csl:247]: I'd do [Epub ahead of print] here as per the style guidelines?

**Review by adam3smith — CHANGES_REQUESTED** [review]
A couple of suggestions/fixes

**Comment by rmzelle** [comment]
Metadata looks fine.

**Comment by POBrien333** [comment]
Thanks for the thorough review. :)

**Pull request merged**